modules/systemd/default.nix

```diff
@@ -41,6 +41,14 @@ with lib; {
         # Link the X11 socket into place. This is a no-op on a normal setup,
         # but helps if /tmp is a tmpfs or mounted from some other location.
         tmpfiles.rules = [ "L /tmp/.X11-unix - - - - ${cfg.wslConf.automount.root}/wslg/.X11-unix" ];
+
+        # Link *.desktop files and icons where WSLg expects them so that GUI apps show up in Start menu
+        tmpfiles.settings.wslg_integration = {
+          "/usr/share/applications".L.argument =
+            "/etc/profiles/per-user/${cfg.defaultUser}/share/applications";
+          "/usr/share/icons".L.argument =
+            "/etc/profiles/per-user/${cfg.defaultUser}/share/icons";
```

Comment:
```suggestion
          "/usr/share/applications".L.argument = "/etc/profiles/per-user/${cfg.defaultUser}/share/applications";
          "/usr/share/icons".L.argument = "/etc/profiles/per-user/${cfg.defaultUser}/share/icons";
```

Reply:
Thanks for suggestion @SuperSandro2000, that's applied now.

Out of curiosity is that some specific editor config? My current setup gave me the formatting from my initial commit and I always found it a bit too eager to fold the longer lines, but I'm new enough with Nix. 😅 My current setup is essentially just default Nix config in Doom Emacs, which isn't ideal, but I haven't seen enough Nix code out there to really question it.

Reply:
That uses nixfmt which by default has the dated ideas about line length. My monitor is wide enough that I don't need to desperately squeeze 80 characters into one line and cribble readability.

Reply:
Yeah keeping the line short by sacrificing readability is not great. Do you use some other formatter?

Reply:
Usually nixpkgs-fmt but adhoc changes done to existing files I usually do by hand to reduce the noise.